web/src/main/java/org/openmrs/module/web/ModuleResourcesServlet.java

```diff
@@ -25,13 +25,13 @@
 import org.slf4j.LoggerFactory;
 
 public class ModuleResourcesServlet extends HttpServlet {
-	
```

Comment:
Can we get rid of most of these formatting changes, so that this is just showing the single line that's changed here.